docs/end-user-topics/dois.rst

```diff
@@ -0,0 +1,91 @@
+
+Dockstore Support for DOIs
+--------------------------
+
+Dockstore can generate DOIs for entries, as well as recognize some DOIs generated outside of Dockstore. Dockstore displays DOIs associated with entries. If you have multiple concept DOIs associated
+with a single entry, you can specify which DOI to display.
+
+Dockstore can generate DOIs for entries 2 ways:
+
+* Explicitly initiated by a Dockstore user. This approach requires you linking your Zenodo account. You select a version via the Dockstore UI and click to create a DOI. See :doc:`Creating Snapshots & Requesting DOIs </advanced-topics/snapshot-and-doi>` for more details.
+* Dockstore can automatically generate DOIs for your entries without you needing to link your Zenodo account. These DOIs are created in the Dockstore Zenodo Community.
+
+In addition, Dockstore can discover DOIs created by a Zenodo-GitHub integration.
+
+Choosing which DOI to display
+-----------------------------
+
+When viewing the public page of an entry, the entry's concept DOI and the selected version's DOI are displayed.
+
+.. figure:: /assets/images/docs/show-doi.png
+   :alt: Show DOIs
+
+Dockstore only displays one DOI associated with a entry, even if there is more than one DOI, e.g., one generated by GitHub-Zenodo integration, and another generated by Dockstore.
+If this situation arises, you can control which DOI is displayed on your dashboard. Select the entry, then click "Manage DOIs", and you'll get a dialog like this. Select your choice
+and click "Save".
+
+   
+.. figure:: /assets/images/docs/manage-dois.png
+   :alt: Manage DOIs
+
+
+Comparison of the different ways of issuing DOIs
+------------------------------------------------
+
+With the different ways Dockstore supports DOIs, it may be confusing as to how to generate DOIs for your Dockstore entries. There are tradeoffs for each method. Here's a high level overview of some of the features of each.
+
+
+Creating Snapshots & Requesting DOIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These DOIs are initiated by entry owners. The entry owner has to explicitly initiate the process, either via the Dockstore UI or the Dockstore API. If issuing a DOI this way, you first need to `snapshot` your version.
+Snapshotting a version makes it immutable, so that no further modifications to the version are allowed by Dockstore.
+
+This approach requires you to have a Zenodo account and link it in Dockstore.
+
+
+* You must initiate the DOI creation
+    * More control over when DOIs are created
+    * More cumbersome, you have to individually do it for every tag
+* Version on Dockstore is snapshotted, which means it will ignore any future changes made on GitHub. This is typically the behavior you want with a tag, but Git tags can be deleted and recreated.
+* Requires a Zenodo account, and that you link your account to Dockstore
```

Comment:
IMHO, this bullet list [and those that follow] mostly reiterate what's already been explained, or include judgements about the utility of the feature, I'm not sure we need them.

Reply:
Reworked